### ISSUE TITLE
[Bug]: 22.0.0 not adapt with appboy-segment-android 12.0.0

### ISSUE THREAD

**Issue by thinhlamzalora** [Appboy/appboy-android-sdk#229]
### Braze Android SDK Version

22.0.0

### Steps To Reproduce

App crash with this line: `use(AppboyIntegration.FACTORY)` in Analytic initialization after update to 22.0.0 because this [issue](https://github.com/Appboy/appboy-android-sdk/issues/225)

### Expected Behavior

App not crash when Analytic is initialized

### Actual Incorrect Behavior

App crash with this line: `use(AppboyIntegration.FACTORY)` in Analytic initialization

### Verbose Logs

```shell
java.lang.VerifyError: Verifier rejected class com.segment.analytics.android.integrations.appboy.AppboyIntegration: com.appboy.IAppboy com.segment.analytics.android.integrations.appboy.AppboyIntegration.getInternalInstance() failed to verify: com.appboy.IAppboy com.segment.analytics.android.integrations.appboy.AppboyIntegration.getInternalInstance(): [0x8]  can't resolve returned type 'Unresolved Reference: com.appboy.IAppboy' or 'Precise Reference: com.braze.Braze'
     com.appboy.Appboy com.segment.analytics.android.integrations.appboy.AppboyIntegration.getUnderlyingInstance() failed to verify: com.appboy.Appboy com.segment.analytics.android.integrations.appboy.AppboyIntegration.getUnderlyingInstance(): [0x8]  can't resolve returned type 'Unresolved Reference: com.appboy.Appboy' or 'Precise Reference: com.braze.Braze' (declaration of 'com.segment.analytics.android.integrations.appboy.AppboyIntegration' appears in /data/app/~~qdUwphO-FqQ2wDnY78jV9g==/com.zalora.dev-eM4IjD_0FDIhzSe4hHlQCw==/base.apk!classes20.dex)
        at pt.rocket.features.tracking.SegmentAnalyticsImpl.init(SegmentAnalyticsImpl.kt:39)
        at pt.rocket.features.tracking.segment.SegmentTracker.init(SegmentTracker.kt:29)
        at pt.rocket.features.tracking.TrackingExecutor$init$1.invoke(TrackingExecutor.kt:20)
        at pt.rocket.features.tracking.TrackingExecutor$init$1.invoke(TrackingExecutor.kt:18)
        at pt.rocket.features.tracking.TrackingUtilsKt$runInBackground$1.invokeSuspend(TrackingUtils.kt:222)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
        at kotlinx.coroutines.internal.LimitedDispatcher.run(LimitedDispatcher.kt:42)
        at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:95)
        at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:570)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:749)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:677)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:664)
    	Suppressed: kotlinx.coroutines.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@d24fef9, Dispatchers.IO]
```

### Additional Information

My app got crash in this line: `.use(AppboyIntegration.FACTORY)`, I think the root cause is Appboy.java has been removed in version 22.0.0

**Comment by chshapiro** [comment]
Hey @thinhlamzalora,

This is expected as the current version of our Segment support Android SDK version 21.0.0. We will shortly be releasing a new version of our Segment SDK which will support Android SDK 22.0.0.

**Comment by thinhlamzalora** [comment]
hi @chshapiro 
Yeah please release soon because our app is facing this [issue](https://github.com/Appboy/appboy-android-sdk/issues/225) , and it cannot be fix until our app update Braze SDK to 22.0.0 but now we cannot update because Braze 22.0.0 not adapt with Segment 12.0.0

**Comment by chshapiro** [comment]
Hey @thinhlamzalora ,

We have released version 13.0.0 of our SDK which supports Braze 22.0.0.

**Comment by thinhlamzalora** [comment]
hi @chshapiro 
Thank you very much, we will check and update you later
